typedb/connection/cluster/user_manager.py

```diff
@@ -75,7 +75,7 @@ def contains(self, username: str) -> bool:
     def get(self, username: str) -> User:
         failsafe_task = _UserManagerFailsafeTask(
             self._client,
-            lambda replica: _ClusterUser.of(self._client._stub(replica.address()).users_get(cluster_user_manager_get_req(username)).get_user(), self._client)
+            lambda replica: _ClusterUser.of(self._client._stub(replica.address()).users_get(cluster_user_manager_get_req(username)).user, self._client)
```

Comment:
This was broken. Protobuf differs to a frightening extent across languages.

Reply:
And this is why we need the tests :D